src/App.js

```diff
@@ -4,9 +4,16 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
 
   render() {
+    const movieListFilter = moviesFromServer.filter(({
```

Comment:
this is not a filter, this is an array, which you get at the end of the calculations, therefore more appropriate name will be `filteredMovies` for example